### PR TITLE
Feature/login and signup pages

### PR DESCRIPTION
**Pull request by Ahontic**

### CODE REVIEW

File: db/migrate/20191217095221_add_confirmable_to_devise.rb [hunk 1]
```diff
@@ -0,0 +1,15 @@
...
+    add_column :customers, :confirmed_at, :datetime
+    add_column :customers, :confirmation_sent_at, :datetime
+    add_index :customers, :confirmation_token, unique: true
+    Customer.update_all confirmed_at: DateTime.now
+  end
+
+  def down
+    remove_columns :customers, :confirmation_token, :confirmed_at, :confirmation_sent_at
+  end
+end
```
Comment (Ahontic) [inline db/migrate/20191217095221_add_confirmable_to_devise.rb]: Screenshot from 2019-12-18 16-15-54
Screenshot from 2019-12-18 16-16-27

File: app/views/books/show.html.haml [hunk 2]
```diff
@@ -28,13 +28,12 @@
...
           %p.h1.in-gold-500.mt-10#book-cost
             = number_to_currency( @book.price, unit: '€')
-          %p.h1.in-gold-500.mt-10.invisible#book-price
-            = number_to_currency( @book.price, unit: '€')
         .col-xs-5.col-md-3
           .input-group.general-position
             %a.input-link{href: "#"}
               %i.fa.fa-minus.line-height-40
             %input.form-control#quantity-input{ value: 1 }
+              %input.form-control.invisible#book-price{ value: "#{@book.price}" }
```
Comment (6aBblKuH) [inline app/views/books/show.html.haml]: use `.none` instead of `.visible`

File: app/views/layouts/_header.html.haml [hunk 3]
```diff
@@ -6,57 +6,38 @@
...
-        = t :title
+      = link_to t(:title), root_path, class: 'navbar-brand'
       %a.shop-link.pull-right.visible-xs{href: "#"}
         %span.shop-icon
           %span.shop-quantity 3
     %a.shop-link.pull-right.hidden-xs{href: "#"}
       %span.shop-icon
         %span.shop-quantity 3
     #navbar.navbar-collapse.collapse.pl-30{'aria-expanded': "false"}
-      .visible-xs
```
Comment (6aBblKuH) [inline app/views/layouts/_header.html.haml]: what about mobile nav?

File: config/environments/development.rb [hunk 4]
```diff
@@ -32,17 +32,19 @@
...
+  config.action_mailer.delivery_method = :letter_opener
   config.action_mailer.perform_caching = false
+  config.action_mailer.perform_deliveries = true
 
   # Print deprecation notices to the Rails logger.
   config.active_support.deprecation = :log
 
   # Raise an error on page load if there are pending migrations.
-  config.active_record.migration_error = :page_load
+  config.active_record.migration_error = false # :page_load
```
Comment (6aBblKuH) [inline config/environments/development.rb]: ?

**Pull request merged**